.gitignore

```diff
@@ -26,16 +26,21 @@
 !/.yarn/sdks
 !/.yarn/versions
 
-/lib/tasks_private/terraform/provision/.terraform
+/lib/tasks_private/terraform/snapshot/terraform.*
+/lib/tasks_private/terraform/snapshot/*.log
+/lib/tasks_private/terraform/snapshot/.terraform
+/lib/tasks_private/terraform/snapshot/.terraform*
 /lib/tasks_private/terraform/provision/terraform.*
 /lib/tasks_private/terraform/provision/*.log
-
-/lib/tasks_private/terraform/placement_group/.terraform
+/lib/tasks_private/terraform/provision/.terraform
+/lib/tasks_private/terraform/provision/.terraform*
 /lib/tasks_private/terraform/placement_group/terraform.*
 /lib/tasks_private/terraform/placement_group/*.log
-
-/lib/tasks_private/terraform/snapshot/.terraform
-/lib/tasks_private/terraform/snapshot/terraform.*
+/lib/tasks_private/terraform/placement_group/.terraform
+/lib/tasks_private/terraform/placement_group/.terraform*
+/lib/tasks_private/terraform/.byebug*
+lib/tasks_private/terraform/snapshot/terraform.*
+/lib/tasks_private/terraform/snapshot/.terraform*
```

Comment:
You could simplify most this using `**`. For example, you can ignore all `.terraform` files/dirs with:
```
/lib/tasks_private/terraform/**/.terraform*
```

There's some good documentation at https://git-scm.com/docs/gitignore#_pattern_format

Reply:
thanks. learnt something new Jay. sure will do it.